pkg/model/bundle.go

```diff
@@ -25,6 +30,7 @@ type BundleDescriptor struct {
 	BundleEntriesFileCount uint64        `json:"count" yaml:"count"`                                     // Number of files which have BundleDescriptor Entries
 	Version                uint64        `json:"version,omitempty" yaml:"version,omitempty"`             // Version for the bundle
 	Deduplication          string        `json:"deduplication,omitempty" yaml:"deduplication,omitempty"` // Type of deduplication used
+	RunStage               string        `json:"runstage,omitempty" yaml:"runstage,omitempty"`           // Path to the run stage
```

Comment:
what is a `RunStage`?

i see that there's also a run stage id as part of an input dataset, yet the run itself might contain data about the semantics of stages only from the `RunFile`, as far as i can tell?

overall, i'm still confused about the meaning of "stages."  it is clear that "run" is a novel, unitary concept.  i'm wanting stages to connect to other internal concepts about runs, and i don't see a clear connection encoded anywhere.

like why is a run made of "stages" rather than "levels?"  what does a stage consist of other than a place within a run that data is used?  wouldn't it be more standard to have a dedicated `RunStage` struct entity type that links here and within `model/run.go` via a unique id?

Reply:
It is important to define the concepts beyond the commends in the structs. Will do so.